src/pages/home/report/ReportActionCompose/ReportActionCompose.tsx

```diff
@@ -559,12 +539,6 @@ function ReportActionCompose({
                             {DeviceCapabilities.canUseTouchScreen() && isMediumScreenWidth ? null : (
                                 <EmojiPickerButton
                                     isDisabled={isBlockedFromConcierge || disabled}
-                                    onModalHide={(isNavigating) => {
-                                        if (isNavigating) {
-                                            return;
-                                        }
-                                        focus();
-                                    }}
```

Comment:
Just curious, does the same problem happen with the emoji picker?

Reply:
Yes, the same one because Composer gets focused after that.